joomla/joomla-cms
# [4.0] Non-clickable module in a menu item

This is a replacement PR for #23609 which resolves the merge conflicts

### To test
npm i

Go the module assignment tab of any menu item
Click on the name of a module
A modal will open
Change the published state of the modal and save and close
Check that the module is now shown with the changed state
Check that there are no js errors in the console
